availability_schedule.py

```diff
@@ -68,14 +68,14 @@ def __validate_form_data(self, errors, **kwargs) -> None:
             # validate timezone
             if key == "timezone" and value:
                 # if the value is null
-                if value == "":
+                if value == "" or value == "None" or value == "Null":
```

Comment:
Actually now I just read the code and it seems we have an `if` branching depending on the `key`. This is not scalable. What if in the future you add like 3 more fields, that means 3 more `elif`.

This is non-blocking but consider using a dictionary that maps a field like `timezone` to a validator function. 

Or better yet, just use a third party validation library like Marshmallow and it will makes your code much shorter and cleaner

Reply:
Can you leave it as an issue, I'll fix them when I have time, for now i'm just going to merge this branch in? Thank you!
@j-ngo828

Reply:
Those are lowest priority at the moment, we need frontend and the AI model the most.

So just merge this in and work on a basic frontend. You can take some design inspiration from devchallenges.io authentication app

Reply:
So the basic REST APIs for all our features are done?

Reply:
nope, remember that this involves an AI model so there will be changes needed to be done to a few endpoints (possibly all endpoints except the ones for authentication).

I highly recommend building a very basic frontend (no need for styling even) to test out the API. There may be hidden issues that won't be uncovered until a client uses the API.